doc/release-notes.md

```diff
@@ -246,6 +238,42 @@ in the Low-level Changes section below.
 
 - See the [Mining](#mining) section for changes to `getblocktemplate`.
 
+`importmulti` RPC method
+------------------------
+
+Error handling in the `importmulti` RPC method has been much improved. All
+input data is validated before attempting to import the scripts and keys.
+
+Imported scripts/addresses are categorised by solvability and spendability:
+
+- A script is *solvable* if the the wallet can construct the txIn to spend outputs
```

Comment:
Not sure txIn saves a lot of space versus readability?

Reply:
agree. Changed txIn -> transaction input.